ekip/nationalparks/management/commands/passes.py

```diff
@@ -105,14 +108,16 @@ def read_pass_list(filename):
 
     with open(filename, 'r', encoding='latin-1') as passcsv:
         field_names = [
-            'name', 'phone', 'city', 'state', 'website', 'annual', 'senior',
-            'access']
+            'name', 'phone', 'city', 'state', 'website', 'annual_senior', 'access']
         passreader = csv.DictReader(
             passcsv, fieldnames=field_names, delimiter=',') 
 
+        header = True
         for l in passreader:
```

Comment:
Calling next() on the iterator here, before the for loop will skip the first row of the CSV instead of doing the header=True, header=False thing.